addons/im_livechat/static/src/core/web/discuss_patch.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.Discuss" t-inherit-mode="extension">
+        <xpath expr="//*[@t-ref='content']" position="attributes">
+            <attribute name="t-if">store.inPublicPage or !(ui.isSmall and ['chat', 'channel', 'livechat'].includes(store.discuss.activeTab))</attribute>
+        </xpath>
+    </t>
+</templates>
```

Comment:
This overrides could be avoided if the condition in template is on `main` and `mailboxes`, so that all other tabs including `livechat` and `whatsapp` are properly taken into account